src/core/lib/security/authorization/rbac_translator.h

```diff
@@ -24,15 +24,13 @@
 
 namespace grpc_core {
 
-struct RbacPolicies {
-  Rbac deny_policy;
-  Rbac allow_policy;
-};
-
-// Translates SDK authorization policy to Envoy RBAC policies. Returns error on
-// failure.
+// Translates SDK authorization policy to Envoy RBAC policies. On success, will
+// return one of the following -
+// 1. One allow RBAC policy or,
```

Comment:
If the deny policy is optional, then how about returning them in this form:

```
struct RbacPolicies {
  absl::optional<Rbac> deny_policy;
  Rbac allow_policy;
};
```

Reply:
Done.